manager/db/changesets/01-init-queues-table.xml

```diff
@@ -4,11 +4,15 @@
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
 
-    <changeSet id="1" author="danxmoran">
+    <changeSet id="1" author="danmoran">
+
         <createTable tableName="queues">
-            <column name="name" type="varchar(250)">
+            <column name="id" type="uuid">
                 <constraints primaryKey="true" />
             </column>
+            <column name="name" type="varchar(250)">
```

Comment:
250 for queue name seems excessive? or are we adding some long unique identifier here?